src/redisraft.h

```diff
@@ -700,6 +701,17 @@ typedef struct JoinLinkState {
     void (*complete_callback)(RaftReq *req);
 } JoinLinkState;
 
+typedef struct MultiState {
+    RaftRedisCommandArray cmds;
+    bool active;
+    bool error;
+} MultiState;
+
+typedef struct ClientState {
+    MultiState multiState;
```

Comment:
Just for style, can you use `multi_state` as variable name.

Reply:
ok